cluster-scope/base/clusterrolebindings/metallb/clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: metallb-allow-privileged-scc
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: allow-privileged
```

Comment:
You can do this without a separate cluster role actually, you can assign `system:openshift:scc:privileged` cluster role here instead.

See, for example:
https://github.com/operate-first/apps/blob/c3e5c619dccc19c44d4de2217f5735d26537f4fc/cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml#L9